permissions/CallLogs.permission

```diff
@@ -0,0 +1,14 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-calllogs
+# x-sailjail-description = Call history
+# x-sailjail-translation-key-long-description = permission-la-calllogs_description
+# x-sailjail-long-description = View and edit previous calls and messages
+
+mkdir     ${HOME}/.local/share/commhistory
+whitelist ${HOME}/.local/share/commhistory
+
+dbus-user.talk org.nemomobile.CommHistory
+dbus-user.broadcast org.nemomobile.CommHistory=org.nemomobile.CommHistory.*@/org/nemomobile/CommHistory
+dbus-user.call org.nemomobile.CommHistory=org.nemomobile.CommHistory.*@/org/nemomobile/CommHistory
```

Comment:
I wonder if any of the other interfaces in org.nemomobile.CommHistory would be useful here. They seem a little disconnected to me and at least for pure call and message history point of view missing those interfaces didn't seem to affect anything.

```
$ /usr/lib/qt5/bin/qdbus org.nemomobile.CommHistory
/
/CommHistoryModel
/org
/org/freedesktop
/org/freedesktop/Telepathy
/org/freedesktop/Telepathy/Client
/org/freedesktop/Telepathy/Client/CommHistory
/org/nemomobile
/org/nemomobile/AccountPresence
/org/nemomobile/CommHistory
```

Those telepathy interfaces actually belong to another service name (but they are exposed through this due to limitations of QDBus) but those /CommHistoryModel and /org/nemomobile/CommHistory make me wonder.

Reply:
There are couple of these "one processes, multiple services" cases that were really difficult to understand from bottom up, errors to introspection point of view as one would need to dig deep into source code in order to grok how it ought to be (which is what should drive these rules) -> what there is about such services is more or less guesswork.

Reply:
Looks like /org/nemomobile/AccountPresence is not in libcommhistory at least but [/CommHistoryModel path is](https://git.sailfishos.org/mer-core/libcommhistory/blob/master/src/constants.h#L31) although it's not really clear if it could be useful on application side ([the only place where it uses the string](https://git.sailfishos.org/mer-core/libcommhistory/blob/master/src/updatesemitter.cpp)).